purchases/src/main/kotlin/com/revenuecat/purchases/amazon/AmazonBilling.kt

```diff
@@ -86,7 +86,7 @@ internal class AmazonBilling(
     constructor(
```

Comment:
The comment above this constructor says it is used via reflection. I couldn't find any reflective usage of it. Would love for someone to confirm that 😄

Reply:
I believe that refers to this: https://github.com/RevenueCat/purchases-android/blob/b8bae4ad69e52ab5c98dcfc1e6d82715eecf4ed5/purchases/src/main/kotlin/com/revenuecat/purchases/BillingFactory.kt#L33. Before, we actually used reflection but we refactored it a while back so the comment seems outdated.